### PR TITLE
Story/dfc 11539 web security module

### PR DESCRIPTION
**Pull request by vallabha24**

### CODE REVIEW

File: DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/Configuration/WebSecurityConfig.config.template [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<webSecurityConfig xmlns:config="urn:telerik:sitefinity:configuration" xmlns:type="urn:telerik:sitefinity:configuration:type" config:version="12.2.7224.0">
+  <httpSecurityHeaders>
+    <responseHeaders>
+      <add value="default-src 'self'; script-src 'self' 'unsafe-inline' 'unsafe-eval' *.googleapis.com *.gstatic.com *.googletagmanager.com *.google-analytics.com apis.google.com __cdnUrl__; style-src 'self' 'unsafe-inline' 'unsafe-eval' __cdnUrl__ *.googleapis.com *.gstatic.com netdna.bootstrapcdn.com kendo.cdn.telerik.com; font-src 'self' __cdnUrl__ kendo.cdn.telerik.com netdna.bootstrapcdn.com; img-src 'self' *.gstatic.com *.googleapis.com *.google-analytics.com __cdnUrl__ https://*.dec.sitefinity.com; media-src 'self' data: blob:; child-src 'self' https://platform.twitter.com/ https://syndication.twitter.com/ https://www.youtube.com/ https://player.vimeo.com/ https://w.soundcloud.com/ apis.google.com accounts.google.com staticxx.facebook.com www.facebook.com web.facebook.com badge.stumbleupon.com; connect-src 'self' accounts.google.com https://*.dec.sitefinity.com *.mktoresp.com;" name="Content-Security-Policy" config:flags="1" />
```
Comment (Muthuramana) [inline DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/Configuration/WebSecurityConfig.config.template]: Double check these, I dont think we need facebook or youtube in here...

File: DFC.Digital/DFC.Digital.Web.Sitefinity/DFC.Digital.Web.Sitefinity.csproj [hunk 2]
```diff
@@ -1169,6 +1170,7 @@
     <Content Include="App_Data\Sitefinity\AddOns\JobProfile\Classifications\Structure\version.sf" />
     <Content Include="App_Data\Sitefinity\AddOns\JobProfile\Classifications\Structure\taxonomies.sf" />
     <Content Include="App_Data\Sitefinity\AddOns\JobProfile\addons-registry.psm1" />
+    <None Include="App_Data\Sitefinity\Configuration\WebSecurityConfig.config.template" />
```
Comment (Muthuramana) [inline DFC.Digital/DFC.Digital.Web.Sitefinity/DFC.Digital.Web.Sitefinity.csproj]: Take this out of csproj

File: DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/Configuration/WebSecurityConfig.config.template (hunk 1 above)
Comment (vallabha24) [reply DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/Configuration/WebSecurityConfig.config.template]: Removed those references

File: DFC.Digital/DFC.Digital.Web.Sitefinity/DFC.Digital.Web.Sitefinity.csproj (hunk 2 above)
Comment (vallabha24) [reply DFC.Digital/DFC.Digital.Web.Sitefinity/DFC.Digital.Web.Sitefinity.csproj]: Removed the template reference

**Pull request merged**